AdminWebsite/AdminWebsite.UnitTests/Services/UserAccountServiceTests.cs

```diff
@@ -111,6 +116,21 @@ public async Task Should_add_JOH_role_to_JOH_user_group()
                 Times.Once);
         }
 
+        [Test]
+        public async Task Should_add_StaffMember_role_to_Staff_Member_user_group()
+        {
+            await _service.AssignParticipantToGroup("staff@hmcts.net", "Staff Member");
+
+            _userApiClient.Verify(
+                x => x.AddUserToGroupAsync(
+                    It.Is<AddUserToGroupRequest>(y => y.GroupName == UserAccountService.External)),
```

Comment:
@alberto-hortiguela it should be Internal Group for Staff members

Reply:
That's done now